### PR TITLE
Get Started doc - issue 3998 fixes

### PR DESCRIPTION
**Pull request by rachelappel**
fixes #4133 
[Internal review url](https://review.docs.microsoft.com/en-us/aspnet/core/getting-started?branch=pr-en-us-4133)

### CODE REVIEW

File: aspnetcore/getting-started.md [hunk 1]
```diff
@@ -24,21 +24,28 @@ uid: getting-started
...
 
-   The `dotnet run` command builds the app first if needed.
+   Change to the `aspnetcoreapp` directory and issue the `dotnet run` command. The `dotnet run` command builds the app first if needed.
 
    ```terminal
+   cd aspnetcoreapp
    dotnet run
    ```
 
 5. Browse to `http://localhost:5000`
```
Comment (Rick-Anderson) [inline aspnetcore/getting-started.md]: remove code fencing so it's clickable - that avoids copy/paste

File: aspnetcore/getting-started.md [hunk 2]
```diff
@@ -24,20 +24,27 @@ uid: getting-started
...
+   cd aspnetcoreapp
    dotnet run
    ```
 
-5. Browse to `http://localhost:5000`
+5. Browse to [http://localhost:5000](http://localhost:5000)
+
+6. Open Pages/Index.cshtml and add the following to the page above the carousel:
+
+```cshtml
```
Comment (Rick-Anderson) [inline aspnetcore/getting-started.md]: Can you add that entire file to the PR and change that to add the highlighted code? (using a file include with highlight and range) You Just show a subset of the index file and add something like
@* Markup removed for brevity *@

**Comment by rachelappel** [comment]
@Rick-Anderson review

File: aspnetcore/getting-started.md [hunk 3]
```diff
@@ -24,20 +24,28 @@ uid: getting-started
...
-   mkdir aspnetcoreapp
-   cd aspnetcoreapp
-   dotnet new web
+   dotnet new razor -o aspnetcoreapp
    ```
     
 4. Run the app.
 
-   The `dotnet run` command builds the app first if needed.
+   Change to the `aspnetcoreapp` directory and issue the `dotnet run` command. The `dotnet run` command builds the app first if needed.
```
Comment (Rick-Anderson) [inline aspnetcore/getting-started.md]: ~Change to the `aspnetcoreapp` directory and issue the `dotnet run` command.~
I don't think most folks need that.

File: aspnetcore/getting-started.md [hunk 4]
```diff
@@ -24,20 +24,28 @@ uid: getting-started
...
+   cd aspnetcoreapp
    dotnet run
    ```
 
-5. Browse to `http://localhost:5000`
+5. Browse to [http://localhost:5000](http://localhost:5000)
+
+6. Open Pages/Index.cshtml and add the following to the page above the carousel:
+
+[!code-html[Main](../../getting-started/sample/getting-started/pages.cshtml?range=18-23)]
```
Comment (Rick-Anderson) [inline aspnetcore/getting-started.md]: NOthing rendered here. See the build.

File: aspnetcore/getting-started.md [hunk 5]
```diff
@@ -24,20 +24,28 @@ uid: getting-started
...
+```cshtml
```
Comment (Rick-Anderson) [inline aspnetcore/getting-started.md]: remove this.

File: aspnetcore/getting-started.md [hunk 6]
```diff
@@ -24,20 +24,28 @@ uid: getting-started
...
-   mkdir aspnetcoreapp
-   cd aspnetcoreapp
-   dotnet new web
+   dotnet new razor -o aspnetcoreapp
    ```
     
 4. Run the app.
 
-   The `dotnet run` command builds the app first if needed.
+    Use the following commands to run the app:
```
Comment (badersur) [inline aspnetcore/getting-started.md]: I think having:

3. Change your current directory to *aspnetcoreapp*
    ```terminal
    cd aspnetcoreapp
    ```

4. Run the app
    ```terminal
    dotnet run
    ```

will be better!

Comment (rachelappel) [reply aspnetcore/getting-started.md]: It was that way before and was requested to change to a shorter format.

Comment (badersur) [reply aspnetcore/getting-started.md]: Sorry, my bad! I didn't see that version :(

**Comment by Rick-Anderson** [comment]
>Open Pages/Index.cshtml and add the following to the page above the carousel:

Edit *Pages/Index.cshtml* and ..
files are italic.
I don't see a carousel. Can you add highlighting? 
`[!code-csharp[Main](start-mvc/sample/src/MvcMovie/Models/MovieDateFormat.cs?name=snippet1&highlight=2,10)]`

**Comment by Rick-Anderson** [comment]
I think this is where it's supposed to go

`
@page
@model IndexModel
@{
    ViewData["Title"] = "Home Page";
}
 ADD CODE HERE and HIGHLIGHT
<div id="myCarousel" class="carousel slide" data-ride="carousel" data-interval="6000">
`

**Comment by rachelappel** [comment]
#sign-off

**Pull request merged**